test/testPost.js

```diff
@@ -0,0 +1,266 @@
+                .post('/api/issues/add')
+                .set('content-type', 'multipart/form-data')
+                .field('title', "de ti titula")
+                .field('text', "de ti tekst")
+                .field('user', 'de ti user')
+                .attach('file', fs.readFileSync('./test/test-large-file.pptx'), 'test-large-file.pptx')
+                .end( (err, res) => {
+                    if(err) console.log(err);
+                    res.should.have.status(400);
+                    res.body.should.be.a('object');
+                    res.body.should.have.property('name').and.to.be.eql('MulterError');
+                    res.body.should.have.property('message').and.to.be.eql('File too large');
+                    res.body.should.have.property('code').and.to.be.eql('LIMIT_FILE_SIZE');
+                    res.body.should.have.property('field').and.to.be.eql('file');
+                    res.body.should.have.property('storageErrors').and.to.be.an('array');
+                    res.body.storageErrors.length.should.be.eql(0);
+                    done()
+                    
+                })
+        })
+
+        it('Post more than 10 files', (done)=>{
+            const file = fs.readFileSync('./test/test-pic.jpg')
+            chai.request(app)
+                .post('/api/issues/add')
+                .set('content-type', 'multipart/form-data')
+                .field('title', "de ti titula")
+                .field('text', "de ti tekst")
+                .field('user', 'de ti user')
+                .attach('file', file, 'test-pic.jpg')
+                .attach('file', file, 'test-pic.jpg')
+                .attach('file', file, 'test-pic.jpg')
+                .attach('file', file, 'test-pic.jpg')
+                .attach('file', file, 'test-pic.jpg')
+                .attach('file', file, 'test-pic.jpg')
+                .attach('file', file, 'test-pic.jpg')
+                .attach('file', file, 'test-pic.jpg')
+                .attach('file', file, 'test-pic.jpg')
+                .attach('file', file, 'test-pic.jpg')
+                .attach('file', file, 'test-pic.jpg')
+                .end( (err, res) => {
+                    if(err) console.log(err);
+                    res.should.have.status(400);
+                    res.body.should.be.a('object');
+                    res.body.should.have.property('name').and.to.be.eql('MulterError');
+                    res.body.should.have.property('message').and.to.be.eql('Unexpected field');
+                    res.body.should.have.property('code').and.to.be.eql('LIMIT_UNEXPECTED_FILE');
+                    res.body.should.have.property('field').and.to.be.eql('file');
+                    res.body.should.have.property('storageErrors').and.to.be.an('array');
+                    res.body.storageErrors.length.should.be.eql(0);
+                    done()
+                    
+                })
+        })
+
+        it('Post two files', (done)=>{
```

Comment:
`It`s  descriptions should describe behaviour, and describes are used for describing state or workflow.
That in mind, your descriptions for `it`'s are good for describes and they should exists, which means that you would need to nest all tests one more lv.
Example:
```
describe('routes', function () {
  describe('POST /api/issues/add', function () {
    describe('successful issue creation', function () {
      it('returns 200 and created issue', function () {
      }
    }
    describe('successful issue creation with two files', function () {
      it('returns 200 and created issue', function () {
      }
    }
    describe('large file', function () {
      it('returns 400 and error message', function () {
      }
    }
```